gtk4/sys/src/lib.rs

```diff
@@ -12717,6 +12783,7 @@ extern "C" {
         menu_model: *mut gio::GMenuModel,
     );
     pub fn gtk_menu_button_set_popover(menu_button: *mut GtkMenuButton, popover: *mut GtkWidget);
+    pub fn gtk_menu_button_set_primary(menu_button: *mut GtkMenuButton, primary: gboolean);
```

Comment:
Another missing since annotation that went unnoticed in the latest review, sorry!

Reply:
fixed